app/src/main/java/org/android/go/sopt/ui/login/LoginActivity.kt

```diff
@@ -0,0 +1,76 @@
+package org.android.go.sopt.ui.login
+
+import android.app.Activity
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.activity.result.ActivityResultLauncher
+import androidx.activity.result.contract.ActivityResultContracts
+import com.google.android.material.snackbar.Snackbar
+import org.android.go.sopt.App
+import org.android.go.sopt.MainActivity
+import org.android.go.sopt.databinding.ActivityLoginBinding
+import org.android.go.sopt.ui.join.JoinActivity
+import org.android.go.sopt.util.User
+import org.android.go.sopt.util.hideKeyboard
+import org.android.go.sopt.util.toast
+
+class LoginActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityLoginBinding
+    private lateinit var activityResultLauncher: ActivityResultLauncher<Intent>
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        binding = ActivityLoginBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        setResultSignUp()
+        initClick()
+    }
+
+    private fun initClick() {
+        binding.loLogin.setOnClickListener {
+            this@LoginActivity.hideKeyboard()
+        }
```

Comment:
```suggestion
        binding.loLogin.setOnClickListener {
            hideKeyboard()
        }
```

setOnClickListener에서 이미 View가 내부 객체로 전달되고 있기 때문에 바로 Activity 확장 함수를 호출해도 될 것 같습니다!